File: Basic_Stack_in_.java
import java.util.Scanner;
import java.util.ArrayList;

class Stack {
    private int size;
    private ArrayList<Integer> stack;

    public Stack(int size) {
        this.size = size;
        stack = new ArrayList<>();
    }

    public boolean isFull() {
        return stack.size() == size;
    }

    public boolean isEmpty() {
        return stack.isEmpty();
    }

    public void push(int item) {
        if (isFull()) {
            System.out.println("Stack is full. Cannot push.");
        } else {
            stack.add(item);
            System.out.println(item + " pushed to the stack.");
        }
    }

    public void pop() {
        if (isEmpty()) {
            System.out.println("Stack is empty. Cannot pop.");
        } else {
            int item = stack.remove(stack.size() - 1);
            System.out.println(item + " popped from the stack.");
        }
    }

    public void display() {
        if (isEmpty()) {
            System.out.println("Stack is empty.");
        } else {
            System.out.print("Stack elements: ");
            for (int item : stack) {
                System.out.print(item + " ");
            }
            System.out.println();
        }
    }

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the size of the stack: ");
        int size = input.nextInt();
        Stack stack = new Stack(size);

        while (true) {
            System.out.println("\nStack Operations:");
            System.out.println("1. Push");
            System.out.println("2. Pop");
            System.out.println("3. Display");
            System.out.println("4. Quit");
            System.out.print("Enter your choice: ");
            int choice = input.nextInt();

            if (choice == 1) {
                System.out.print("Enter the item to push: ");
                int item = input.nextInt();
                stack.push(item);
            } else if (choice == 2) {
                stack.pop();
            } else if (choice == 3) {
                stack.display();
            } else if (choice == 4) {
                System.out.println("Exiting the program.");
                break;
            } else {
                System.out.println("Invalid choice. Please try again.");
            }
        }
    }
}

File: Basic_Stack_in.py
class Stack:
    def __init__(self, size):
        self.size = size
        self.stack = []
        
    def isFull(self):
        return len(self.stack) == self.size
    
    def isEmpty(self):
        return len(self.stack) == 0
    
    def push(self, item):
        if self.isFull():
            print("Stack is full. Cannot push.")
        else:
            self.stack.append(item)
            print(f"{item} pushed to the stack.")
    
    def pop(self):
        if self.isEmpty():
            print("Stack is empty. Cannot pop.")
        else:
            item = self.stack.pop()
            print(f"{item} popped from the stack.")
    
    def display(self):
        if self.isEmpty():
            print("Stack is empty.")
        else:
            print("Stack elements:", end=" ")
            for item in self.stack:
                print(item, end=" ")
            print()

size = int(input("Enter the size of the stack: "))
stack = Stack(size)

while True:
    print("\nStack Operations:")
    print("1. Push")
    print("2. Pop")
    print("3. Display")
    print("4. Quit")
    choice = int(input("Enter your choice: "))
    
    if choice == 1:
        item = int(input("Enter the item to push: "))
        stack.push(item)
    elif choice == 2:
        stack.pop()
    elif choice == 3:
        stack.display()
    elif choice == 4:
        print("Exiting the program.")
        break
    else:
        print("Invalid choice. Please try again.")
